internal-services/catalog/iib-add-fbc-fragment-to-index-image-task.yaml

```diff
@@ -93,26 +134,18 @@ spec:
           "binary_image": "$(params.binaryImage)",
           "build_tags": `echo $(params.buildTags)`,
           "add_arches": `echo $(params.addArches)`,
-          "overwrite_from_index": $(params.overwriteFromIndex),
+          "overwrite_from_index": ${FBCOptIn},
           "overwrite_from_index_token": "${IIB_OVERWRITE_FROM_INDEX_USERNAME}:${IIB_OVERWRITE_FROM_INDEX_TOKEN}"
```

Comment:
does IIB complain if you give a token but overwrite is `false`?

Reply:
Yes. It does. So the next part of the script has a json that checks if the value is false and remove both parameters.